diary-alexander-jaffray/modernized/32-ch-16-scotland.adoc

```diff
@@ -267,6 +272,9 @@ It affords some clear indications of a sound and living faith,
 an unsophisticated simplicity of love, a heavenly meekness and gratitude,
 which were now fast ripening for a better state.
 
+[.embedded-content-document.epistle]
```

Comment:
Ok so this first paragraph here is embdded in the ODT and looks like it should be in the PDF as well, but doesn't seem to really be part of the epistle. I wonder if it was just a quote from the "Friends of Aberdeen" that didn't get ended before they inserted the epistle? Or is it taken from some other writing as part of the epistle? I just can't tell from any of the context either. I tried to attach a picture, but it didn't seem to work. It's on page 359 in the PDF and page 243 of the ODT.